electron-builder.json

```diff
@@ -0,0 +1,91 @@
+{
+  "productName": "Baiyang-lzy's Blog Generator",
+  "appId": "com.github.baiyang-lzy.bbg",
+  "asar": true,
+  "files": [
+    "dist/electron/**/*",
+    "./main.js",
+    "App/*"
```

Comment:
`App/*` doesn't match all files under `App/`, such as `App/blog_source`.

https://www.electron.build/file-patterns#multiple-glob-patterns

Reply:
Resolved in [5ebb156](https://github.com/baiyang-lzy/bbg/commit/5ebb156fba0a917a701e682e6b0c00aabb3193a7).